### DISCUSSION TITLE
[OP Stack Deployment] Failing to deploy Create2 Factory on L1 (Due to EIP-155)

### DISCUSSION THREAD

**Discussion by yerasyla** [ethereum-optimism/developers#456] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I am following Deploy the Create2 Factory (Optional) docs: https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#deploy-the-create2-factory-optional

I am planning to deploy OPStack chain on custom L1 with EIP-155 upgrade enabled. I've already tried --rpc.allow-unprotected-txs=true and other methods. But still transaction is blocked.

I believe there is only 2 option: modifying L1 chain code, deploying Create2 with custom address. What can you suggest?

**Comment by sbvegan** [comment]
Hey @yerasyla, what L1 are you using? If eip-155 is enabled on the protocol level this might be hard to circumvent. My first advice is what you already tried, using a L1 node with `--rpc.allow-unprotected-txs=true`. If you're running your own custom L1, then you'd likely need to modify the L1 code. This isn't something we can address on the OP Stack side because you're at the mercy of the L1's rules.
